Directory.Packages.props

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <!-- Akka.NET Package Versions -->
   <ItemGroup>
+    <PackageVersion Include="Akka" Version="1.5.20" />
```

Comment:
To make sure we avoid https://github.com/akkadotnet/akka.net/pull/7174 until Akka.Hosting v1.5.20 ships.